### PR TITLE
Task 86296: check-mark

### PR DESCRIPTION
**Pull request by Lisa-V-Sau**
Cypress test includes:

-  document Uncheck

- document Checked

### CODE REVIEW

**Review by CathLass — COMMENTED** [review]
It looks like you're assuming that the box will always be checked when you start these tests? We don't have stable data so that can't be guaranteed.

**Comment by Lisa-V-Sau** [comment]
Yes, that did cross my mind. We don’t have another option till we get a staging environment as such.

The test checks and unchecks the box, therefore would it help if we ask everyone to also be mindful when exploring the site? I.e., to set the status back from how they found it?

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/84457_TL_school-and-trust-information-proj-dates/86296_error-handling_checkmark.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    before(function () {
+        cy.login();
+        cy.selectSchoolListing(1)
+    });
+
+    it("TC: Precondition checkbox status", () => {
+        cy.get('[id=school-and-trust-information-status]').should("be.visible")
+        .invoke("text")
+        .then((text) => {
+            // text == "Completed" || "In Progress"
```
Comment (CathLass) [inline ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/84457_TL_school-and-trust-information-proj-dates/86296_error-handling_checkmark.js:16]: can we get rid of the commented-out line now

Comment (CathLass) [reply ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/84457_TL_school-and-trust-information-proj-dates/86296_error-handling_checkmark.js:16]: ... and the cy.log if you don't need it any more

**Review by CathLass — CHANGES_REQUESTED** [review]

**Review by CathLass — APPROVED** [review]

**Pull request merged**